rotkehlchen/chain/ethereum/eth2.py

```diff
@@ -93,8 +95,8 @@ def deserialize_from_db(
             timestamp=Timestamp(int(deposit_tuple[3])),
             pubkey=deposit_tuple[4],
             withdrawal_credentials=deposit_tuple[5],
-            value=Balance(amount=FVal(deposit_tuple[6])),
```

Comment:
@vnavascues the bug was essentially that you just serialized/deserialized the amount of ether (so the amount part of the balance) and not the USD part.

Make sure to always serialize everything in the future (unless we specify we don't need something).

I also completely missed it during review of the PR :sweat_smile: